AtelierTomato.Markov/AtelierTomato.Markov.Data.Test/MultiParserTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using AtelierTomato.Markov.Data.Model;
+using AtelierTomato.Markov.Data.Model.ObjectOID;
+using AtelierTomato.Markov.Data.Model.ObjectOID.Parser;
+using FluentAssertions;
+
+namespace AtelierTomato.Markov.Data.Test
+{
+	public class MultiParserTest
+	{
+		[Fact]
+		public void DiscordIObjectOIDParseTest()
+		{
+			MultiParser<IObjectOID> oidParser = new([new BookObjectOIDParser(), new InvalidObjectOIDParser(), new DiscordObjectOIDParser()]);
+			IObjectOID OID = oidParser.Parse("Discord:discord.com:1253189664655806606:1253189664655806610");
+			OID.Should().BeEquivalentTo(DiscordObjectOID.Parse("Discord:discord.com:1253189664655806606:1253189664655806610"));

```

Comment:
I would also check for expected return type here (that is, DiscordObjectOid), same in equivalent tests